app/controllers/users_controller.rb

```diff
@@ -158,6 +159,30 @@ def custom_ontologies
     redirect_to user_path(@user.username)
   end
 
+  
```

Comment:
I see that the two methods `subscribe` and  `un_subscribe` share some code, extract it

Reply:
which part of the code is duplicated plz?